app/components/footer/footer.component.js

```diff
@@ -0,0 +1,5 @@
+import template from "./footer.template.jade";
```

Comment:
according [style guide](https://github.com/toddmotto/angular-styleguide) it should be footer.jade